doc/classes/CheckButton.xml

```diff
@@ -1,10 +1,10 @@
 <?xml version="1.0" encoding="UTF-8" ?>
 <class name="CheckButton" inherits="Button" version="4.1" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../class.xsd">
 	<brief_description>
-		Checkable button. See also [CheckBox].
+		A button that represents a binary choice.
 	</brief_description>
 	<description>
-		CheckButton is a toggle button displayed as a check field. It's similar to [CheckBox] in functionality, but it has a different appearance. To follow established UX patterns, it's recommended to use CheckButton when toggling it has an [b]immediate[/b] effect on something. For example, it could be used if toggling it enables/disables a setting without requiring the user to press a confirmation button.
+		[CheckButton] is a toggle button displayed as a check field. It's similar to [CheckBox] in functionality, but it has a different appearance. To follow established UX patterns, it's recommended to use [CheckButton] when toggling it has an [b]immediate[/b] effect on something. For example, it could be used if toggling does something immediately, without requiring the user to press a confirmation button.
```

Comment:
"it has immediate effect, e.g. does something immediately"
I think a more concrete example is needed 🤔

Reply:
Oops, how embarrassing, I think I had a specific example but scrapped it and left it like this. I'll go with a "shows or hides advanced settings" example.